addons/website/views/snippets/s_numbers_boxed.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_boxed" name="Numbers boxed">
+    <section class="s_numbers_boxed pt80 pb80 o_colored_level o_cc o_cc5">
+        <div class="container">
+            <h2 style="text-align: center;">Insights, stats, and metrics</h2>
+            <p style="text-align: center;">This section allows to highlight key statistics.</p>
+            <div class="row o_grid_mode" data-row-count="6" style="column-gap: 40px;">
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-12 col-lg-4 border rounded o_cc o_cc3" style="grid-area: 2 / 1 / 7 / 5; --grid-item-padding-y: 64px; text-align: center; border-radius: 9.6px !important;">
+                    <h3 class="display-3"><font style="background-image: linear-gradient(135deg, var(--o-color-4) 15%, var(--o-color-5) 100%);" class="text-gradient">30+</font></h3>
+                    <p class="fw-bold">Useful options</p>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-4 border rounded o_cc o_cc3" style="grid-area: 2 / 5 / 7 / 9; --grid-item-padding-y: 64px; text-align: center; border-radius: 9.6px !important;">
+                    <h3 class="display-3"><font style="background-image: linear-gradient(45deg, var(--o-color-4) 15%, var(--o-color-5) 100%);" class="text-gradient">45%</font></h3>
+                    <p class="fw-bold">More leads</p>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-4 border rounded o_cc o_cc3" style="grid-area: 2 / 9 / 7 / 13; --grid-item-padding-y: 64px; text-align: center; border-radius: 9.6px !important;">
+                    <h3 class="display-3"><font style="background-image: linear-gradient(45deg, var(--o-color-4) 15%, var(--o-color-5) 100%);" class="text-gradient">8+</font></h3>
+                    <p class="fw-bold">Amazing pages</p>
```

Comment:
I don't know the `fw-bold` 🤔 I see that the build wrap bold text with a `<strong>` tag so I guess it has to be done like that.